fulp_modules/features/clothing/body_cameras/body_camera.dm

```diff
@@ -41,11 +41,18 @@
 		return FALSE
 	return TRUE
 
-/obj/item/bodycam_upgrade/proc/turn_on(mob/user, obj/item/card/id/id_card)
+/obj/item/bodycam_upgrade/proc/turn_on(mob/living/user, obj/item/card/id/id_card)
+	if(!id_card)
+		var/obj/item/card/id/card = user.get_idcard()
+		if(card)
+			id_card = card
 	builtin_bodycamera = new(user)
 	builtin_bodycamera.internal_light = FALSE
 	builtin_bodycamera.network = list("ss13")
-	builtin_bodycamera.c_tag = "-Body Camera: [(id_card.registered_name)] ([id_card.assignment])"
+	if(id_card)
+		builtin_bodycamera.c_tag = "-Body Camera: [(id_card.registered_name)] ([id_card.assignment])"
+	else
+		builtin_bodycamera.c_tag = "-Body Camera: [(user.name)]"
```

Comment:
This isn't a problem with our pop and players but on the old fulp repo with high pop people were able to just put a bunch of body cams on people to flood the camera list.

Reply:
That sounds like a large amount of effort for a very minimal effect if I'll be honest. Does that exploit do anything aside from making it necessary to scroll just a bit more?